### PR TITLE
Update ZBOSS libraries to release v3.9.0.0 && remove Zigbee GP Combo Basic libraries

### PR DESCRIPTION
**Pull request by sebastiandraus**

### CODE REVIEW

**Review by greg-fer — APPROVED** [review]

**Comment by sebastiandraus** [comment]
Force push to re-trigger CI

File: doc/nrf/releases/release-notes-changelog.rst [hunk 1]
```diff
@@ -334,6 +334,9 @@ Libraries for Zigbee
 * Fixes and improvements in :ref:`Zigbee Shell  <lib_zigbee_shell>` library.
 * Added :ref:`BDB command for printing install codes <bdb_ic_list>` to the :ref:`Zigbee shell <lib_zigbee_shell>` library.
 * Improve logging in :ref:`ZBOSS OSIF <lib_zigbee_osif>` library and :ref:`Zigbee Shell <lib_zigbee_shell>` library.
+* Removed support for Zigbee Green Power Proxy Basic functionality.
```
Comment (tomchy) [inline doc/nrf/releases/release-notes-changelog.rst:337]: Incorrect!
Should be:
Removed experimental support for Green Power Combo Basic functionality.

File: doc/nrf/releases/release-notes-changelog.rst [hunk 2]
```diff
@@ -334,6 +334,9 @@ Libraries for Zigbee
...
+* Updated ZBOSS Zigbee stack to version v3.9.0.0+v4.0.1.
```
Comment (tomchy) [inline doc/nrf/releases/release-notes-changelog.rst:338]: Wrong version code

**Review by tomchy — CHANGES_REQUESTED** [review]

File: doc/nrf/releases/release-notes-changelog.rst (hunk 1 above)
Comment (sebastiandraus) [reply doc/nrf/releases/release-notes-changelog.rst]: done!

File: doc/nrf/releases/release-notes-changelog.rst (hunk 2 above)
Comment (sebastiandraus) [reply doc/nrf/releases/release-notes-changelog.rst]: done!

**Review by tomchy — APPROVED** [review]

**Comment by sebastiandraus** [comment]
PR in nrfxlib rebased at newest main - re-triggering CI

File: west.yml [hunk 3]
```diff
@@ -104,7 +104,7 @@ manifest:
     - name: nrfxlib
       repo-path: sdk-nrfxlib
       path: nrfxlib
-      revision: 22d60f831a43f988fbb9c4475dd4a5ae657930aa
+      revision: pull/580/head
```
Comment (rlubos) [inline west.yml:107]: Please update SHA: `f3677ffd5d4599ef1f521340717c347694677123`

**Review by rlubos — APPROVED** [review]

**Review by greg-fer — CHANGES_REQUESTED** [review]
Please reflect the change to ZBOSS 3.9.0.1 in the release notes for ZBOSS. Currently, 3.9.0.0 is listed there.

**Review by greg-fer — APPROVED** [review]
Removing the request changes, as Sebastian is about to deploy fix to the docs. I'll then update links in #6074 .

**Pull request merged**